### ISSUE TITLE
Allow live links

### ISSUE THREAD

**Issue by jakebeal** [BDAthlon/2017-Triple_Helix-1#2]
When glyphs are updated in their master version, the repository should automatically update as well.
For example, we're adopting a bunch of new stuff in SBOLv, and it would be good to have those track the current master branch in GitHub rather than end up with version-skew when things get updated in GitHub but not necessarily in your repository.

**Comment by jamesscottbrown** [comment]
I'm not sure I understand: [How to Propose a New SBOL Glyph](https://docs.google.com/document/d/1hPuvkAQY0pU3xRNMNWMMq7wDGqanYes4SF4CSjuugBU) document says that once a glyph has been Endorsed "the glyph should not generally be changed", and glyphs aren't added to the GitHub repository before then.

Are you suggesting that glyphs added to the [SBOLv-realizations repository](https://github.com/SynBioDex/SBOLv-realizations) should be automatically imported?

**Comment by jakebeal** [comment]
While there is not likely to be much change, I do expect that there will be some, as implementations get tweaked over time --- things that aren't really "changing a glyph" but cleaning up aspects of its canonical form.

So I don't think automatic import should happen, but I think that an import should track a linked version (which might be an updatable "head") rather than taking a static copy.

**Comment by jamesscottbrown** [comment]
I agree that relying on people to manually keep things in sync is not ideal (and the various formats of SO-SBOLv mapping table provide an illustration).

However, I'm concerned that hot-linking to image files in a branch of the [SBOLv-realizations GitHub repository](https://github.com/SynBioDex/SBOLv-realizations) (if that is what you are suggesting) would introduce a brittle dependency. If either the structure of this repository were to change, or GitHub were to change its URL scheme for accessing a raw file in a branch, then all of the hot-linked glyphs would stop being displayed. Additionally, the images would be unavailable during temporary GitHub outages.

I think that a better solution might be for me to write a script that would automatically copy files from the GitHub repository to the appropriate locations on the glyph web server. This would allow updates to glyphs to be propogated without having to manually uploading each individual modified file, whilst also ensuring that images of submitted glyphs would continue to be served regardless of what happens to the SBOLv-realizations repository.

**Comment by jakebeal** [comment]
I think you and I are thinking along the same lines: have a local instance --- in fact, that local instance could even be backed by a git repository cloned from the GitHub source. So then you get both resilience and automatic updates.
